docs/guidelines/secret-scanning.md

```diff
@@ -0,0 +1,118 @@
+# Secret Scanning
+
+As developers, we know that secrets like passwords, API keys, and access tokens are critical to our work. But what happens when these secrets accidentally end up in our code, logs or error messages?
+
+This guideline aims to present relevant tools, some good practices for managing this risk, and what to do when we have messed up. The AppSec team provides a 4 hour hands-on workshop to get started with secret scanning.
+
+- [Recommended tools](#recommended-tools)
+- [Where to scan for secrets in our SDLC](#where-to-scan-for-secrets-in-our-sdlc)
+- [What to do when we have messed up](#what-to-do-when-we-have-messed-up)
+- [How do we manage secrets in our dev environments?](#how-do-we-manage-secrets-in-our-dev-environments)
+
+## Recommended tools
```

Comment:
We have another place for tools as well here - appsec/resources/tools/

May be mention these there as well ? or remove that section  to avoid confusion ?

Reply:
I'll add a point to these as well.